### PR TITLE
Adds `arrowDirection` prop

### PR DESCRIPTION
**Pull request by imartingraham**
This adds the ability to declare where the ToolTip displays

### CODE REVIEW

File: ToolTip.ios.js [hunk 1]
```diff
@@ -16,13 +16,14 @@ var propTypes = {
...
   })),
+  arrowDirection: React.PropTypes.string,
   longPress: React.PropTypes.bool,
   ...TouchableHighlight.propTypes,
 };
 
 var ViewClass = React.createClass({
   showMenu: function() {
-    ToolTipMenu.show(findNodeHandle(this.refs.toolTipText), this.getOptionTexts());
+    ToolTipMenu.show(findNodeHandle(this.refs.toolTipText), this.getOptionTexts(), this.props.arrowDirection || 'down');
```
Comment (jrichardlai) [inline ToolTip.ios.js]: you can use `defaultProps` for this

File: ToolTip.ios.js [hunk 2]
```diff
@@ -16,13 +16,14 @@ var propTypes = {
     text: React.PropTypes.string.isRequired,
     onPress: React.PropTypes.func,
   })),
+  arrowDirection: React.PropTypes.string,
```
Comment (jrichardlai) [inline ToolTip.ios.js]: can you make this validate that its one of the options?

**Comment by jrichardlai** [comment]
Thanks for the PR, can you update the doc too?

**Comment by imartingraham** [comment]
@jrichardlai Updated the documentation, PropType declaration and set the defaultProp value for arrowDirection

File: ToolTip.ios.js [hunk 3]
```diff
@@ -16,13 +16,20 @@ var propTypes = {
...
     onPress: React.PropTypes.func,
   })),
+  arrowDirection: React.PropTypes.oneOf(['up', 'down', 'left', 'right']),
   longPress: React.PropTypes.bool,
   ...TouchableHighlight.propTypes,
 };
 
 var ViewClass = React.createClass({
+  getDefaultProps: function() {
+      return {
```
Comment (jrichardlai) [inline ToolTip.ios.js]: Can you use 2 spaces here :)!

Comment (imartingraham) [inline ToolTip.ios.js]: All fixed!

**Comment by jrichardlai** [comment]
Thanks!

**Pull request merged**